docs/omnipaxos/features.md

```diff
@@ -1,8 +1,9 @@
 OmniPaxos provide several features that can be used to enhance both usability and performance:
 
+- `persistent_storage` - RocksDB-backed storage implementation for OmniPaxos state. Allows for recovery even if all nodes in a cluster fail.
```

Comment:
Perhaps we can add another header for features that are specific to `omnipaxos_storage`. To avoid confusion since `persistent_storage` is not a feature flag for omnipaxos.